### ISSUE TITLE
Use a consistent template for PPT materials

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-marketing#64]
For tutorial material that is tied directly to the standard, for example the explainer and perhaps a tutorial on the TD, we should use a consistent PPT template.   Perhaps we can use the template from the AC meeting as a starting point, and add the WoT logo...

This can be used for other material besides tutorials as well.   A consistent look and feel will help build the brand.

**Comment by egekorkan** [comment]
I would also suggest agreeing on a slide to put at the end which has links to multiple WoT pages, tools etc.

**Comment by k-toumura** [comment]
@ashimura, Is there any authoring tool for "AC meeting" style presentation?

**Comment by mmccool** [comment]
I agree, it would be best to start with a "standard" W3C template, THEN add WoT-specific customizations (eg the WoT logo).  It would also be nice to do this "properly" as a real PPT template (so the logo is in the template, not something you have to cut and paste, for instance...).

**Comment by mmccool** [comment]
I started a draft... did not get any info on a standard W3C template so I started from a simple basic template.  See PR https://github.com/w3c/wot-marketing/pull/67 .

**Comment by danielpeintner** [comment]
I wonder whether the slides need to mention any *legal* statements in one way or the other.
@ashimura ?

**Comment by egekorkan** [comment]
Call of 12.04:
- We do not need legal statements since we are presenting it as our presentation unless the presenter is a W3C Staff or if you want to affiliate the presentation with W3C
- We as the IG/WG should define our own policy and then members can decide for themselves.

- @egekorkan will continue next time about having more reusable content in the template. @mmccool's slides for various can be used, as well as others
